js/index.js

```diff
@@ -40,3 +40,110 @@ const siteContent = {
+nav[0].textContent = siteContent ["nav"] ["nav-item-1"];
+nav[1].textContent = siteContent ["nav"] ["nav-item-2"];
+nav[2].textContent = siteContent ["nav"] ["nav-item-3"];
+nav[3].textContent = siteContent ["nav"] ["nav-item-4"];
+nav[4].textContent = siteContent ["nav"] ["nav-item-5"];
+nav[5].textContent = siteContent ["nav"] ["nav-item-6"];
+
+
+const textHeading = document.querySelectorAll('h4');
+textHeading[0].textContent = siteContent ["main-content"] ["features-h4"];
+textHeading[1].textContent = siteContent ["main-content"] ["about-h4"];
+textHeading[2].textContent = siteContent ["main-content"] ["services-h4"];
+textHeading[3].textContent = siteContent ["main-content"] ["product-h4"];
+textHeading[4].textContent = siteContent ["main-content"] ["vision-h4"];
+
+const pContent = document.querySelectorAll('p');
+pContent[0].textContent = siteContent ["main-content"] ["features-content"];
+pContent[1].textContent = siteContent ["main-content"] ["about-content"];
+pContent[2].textContent = siteContent ["main-content"] ["services-content"];
+pContent[3].textContent = siteContent ["main-content"] ["product-content"];
+pContent[4].textContent = siteContent ["main-content"] ["vision-content"];
+
+const contactH = document.querySelector('.contact h4');
+contactH.textContent = siteContent ["contact"] ["contact-h4"];
+
+const contactP = document.querySelectorAll('.contact p');
+contactP[0].textContent = siteContent ["contact"] ["address"];
+contactP[1].textContent = siteContent ["contact"] ["phone"];
+contactP[2].textContent = siteContent ["contact"] ["email"];
+
+const footer = document.querySelector('footer p');
+footer.textContent = siteContent ["footer"] ["copyright"];
+
+
+const callText = document.querySelector(".cta-text h1");
+callText.textContent = "Dom is Awesome";
+
+let btn = document.querySelector('.cta-text button');
+btn.textContent = "Get Started";
+
+let cta = document.querySelector('.cta');
+
+let codePic = document.getElementById("cta-img");
+codePic.setAttribute('src', siteContent["cta"]["img-src"])
+btn.addEventListener('click', (event) => {
+  // codePic.style.position = 'absolute';
+  codePic.style.visibility = 'hidden';
+  cta.style.visibility = 'hidden';
+  mainContent.style.visibility = 'hidden';
+
+})
+
+let midPic = document.getElementById("middle-img");
+midPic.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+midPic.style.width = '800px';
+midPic.style.borderRadius = '20px';
+
+
+
+
+
+// ## Task 3: Add new content
+// * [ ] Change the color of the navigation text to be green.
+// * [ ] Utilize `.appendChild()` and `.prepend()` to add two new items to the navigation system. You can call them whatever you want.
+// * [ ] Check your work by looking at the [original html](original.html) in the browser
+
+nav[0].style.color = 'green';
+nav[1].style.color = 'green';
+nav[2].style.color = 'green';
+nav[3].style.color = 'green';
+nav[4].style.color = 'green';
+nav[5].style.color = 'green';
```

Comment:
In terms of organization - try to keep like things together, and allow the page to be read top to bottom as the webpage would be